source/Octopus.TestPortForwarder/Octopus.TestPortForwarder.csproj

```diff
@@ -8,10 +8,11 @@
     </PropertyGroup>
 
     <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
+        <TargetFrameworks>net48;net6.0;net8.0</TargetFrameworks>
```

Comment:
Double up

Reply:
net6.0 and net8.0 aren't double-ups?

Reply:
![image](https://github.com/OctopusDeploy/Halibut/assets/86938706/e6ee0d5b-2b2e-40c3-b52f-5c4497995fa0)